Added support for dynamic events.

melomenic opened the pull request:
Support added for dynamic events introduced in .NET 4.6. Since the project is currently compiled against 4.5, the payload names are not made available in the schema at this time. A warning has been added indicating what needs to be done when the project is upgraded to 4.6 to support payload names. This addresses issue #44, and partially addresses #64.

Commit:
Update ObservableEventListener.cs

Commit:
Update EventSourceSchemaReader.cs

Commit:
Update EventSourceSchemaReader.cs

Commit:
Update EventSourceSchemaReader.cs

msftclas commented:
Hi __@melomenic__, I'm your friendly neighborhood Microsoft Pull Request Bot (You can call me MSBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.microsoft.com.
    </span>

TTYL, MSBOT;

azurecla commented:
__@melomenic__, Thanks for signing the contribution license agreement so quickly! Actual humans will now validate the agreement and then evaluate the PR.
<br />Thanks, AZPRBOT;

msftclas commented:
__@melomenic__, Thanks for signing the contribution license agreement so quickly! Actual humans will now validate the agreement and then evaluate the PR.
<br />Thanks, MSBOT;

Commit:
Update EventSchema.cs

Added ability to set event name.

Commit:
Update EventSourceSchemaReader.cs

Updated GetDynamicSchema(). Added support for event name when project is converted to .NET 4.6.

bennage commented:
Please see the note on #91. We are no longer working on Semantic Logging.
